src/bp/ui-studio/src/web/views/FlowBuilder/inspector/index.tsx

```diff
@@ -40,9 +41,19 @@ interface Props {
 
 class Inspector extends Component<Props> {
   render() {
+    const goBackToMain = () => {
+      this.props.closeFlowNodeProps()
+      this.props.refreshFlowsLinks()
+    }
+
     const node = this.props.currentFlowNode
     return (
       <div className={style.inspector}>
+        {node && (
+          <Button id="btn-back-element" className={style.noLineHeight} onClick={goBackToMain} bsSize="xsmall">
+            <i className="material-icons">keyboard_backspace</i>
+          </Button>
```

Comment:
We are trying to get rid of react-bootstrap and reactstrap, please use the Blueprint library when possible:
https://blueprintjs.com/docs/#core/components/button

Reply:
Ah. my bad, saw it in another component and thought there was a reason for it